src/app/child-dev-project/notes/notes-related-to-entity/notes-related-to-entity.component.ts

```diff
@@ -89,17 +92,64 @@ export class NotesRelatedToEntityComponent implements OnInit {
       } else if (this.entity.getType() === ChildSchoolRelation.ENTITY_TYPE) {
         newNote.addChild((this.entity as ChildSchoolRelation).childId);
         newNote.addSchool((this.entity as ChildSchoolRelation).schoolId);
-        newNote.relatedEntities.push(this.entity.getId(true));
-      } else {
-        newNote.relatedEntities.push(this.entity.getId(true));
       }
 
+      newNote.relatedEntities.push(this.entity.getId(true));
+      this.getIndirectlyRelatedEntityIds(this.entity).forEach((e) =>
+        newNote.relatedEntities.push(e),
+      );
```

Comment:
This "bloats up" the index slightly: If relatedEntities are enabled (by default they are disabled, having `additional: undefined`), child or school ids may end up in both relatedEntities and children/schools property. The Service is removing duplicate results when loading the query however, so functionally this shouldn't become an issue.